cms/djangoapps/contentstore/utils.py

```diff
@@ -83,7 +83,12 @@
 from xmodule.modulestore.django import modulestore  # lint-amnesty, pylint: disable=wrong-import-order
 from xmodule.modulestore.exceptions import ItemNotFoundError  # lint-amnesty, pylint: disable=wrong-import-order
 from xmodule.partitions.partitions_service import get_all_partitions_for_course  # lint-amnesty, pylint: disable=wrong-import-order
-from xmodule.services import SettingsService, ConfigurationService, TeamsConfigurationService
+from xmodule.services import (
+    SettingsService,
+    ConfigurationService,
+    TeamsConfigurationService,
+    ResultService
```

Comment:
I'm worried that "ResultService" might imply that it's involved with the scoring. FWIW, I agree with @Agrendalath that we don't need to have separate services for correctness and answer, but I'm not convinced about the naming. Maybe `ProblemFeedbackService`?